src/foundry/common/types.mjs.d.ts

```diff
@@ -1,3 +1,86 @@
+interface DocumentModificationOptions {
```

Comment:
The type `DocumentModificationOptions` technically does not exist in foundry. I know that it's probably still used in a lot of places, so we kind of need it, but that's the reason it wasn't global so far. I guess we should probably still keep it and make it global, since it's used so much....

Reply:
I've got a thought, convert it to a type, using `Exclude<DocumentModificationContext, 'parent' | 'pack'>` to pull off things not present. This way `DocumentModificationContext` follows core and we still have our internal type. `DocumentModificationOptions` then could be moved back to `document.mjs.d.ts` or something as an export. Thoughts?

Reply:
Sounds good to me! 👍